experimental/ExteriorAlgebra/ExteriorAlgebra_naive.jl

```diff
@@ -0,0 +1,95 @@
+export exterior_algebra_naive;  # constructor
+export ContainsDuplicates;   # export just for developing/testing
+
+# Auxiliary function -- worth keeping?
+
+function ContainsDuplicates(L)  # L must be indexable (1:n), and its elems must be sortable
```

Comment:
Function names should be snake case; i.e. this should be `contains_duplicates`.

Reply:
Thanks!  It seems that the Julia function `allunique` actually does what I need, so `ContainsDuplicates` will be deleted.